src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag

```diff
@@ -32,6 +32,16 @@
         Upload/Edit Photo
       </button>
     </div>
+    <br>
+    <br>
+    <div class="">
+      <button id="deletePhoto"
```

Comment:
`delete-photo`

Reply:
@wkurniawan07 Id of Upload/Edit button is <button id="uploadEditPhoto", shouldn't we follow same pattern?